client/app/stores/account_store.coffee

```diff
@@ -141,6 +141,7 @@ class AccountStore extends Store
 
         handle ActionTypes.NEW_ACCOUNT_ERROR, (error) ->
             _newAccountWaiting = false
+            error.uniq = Math.random()
```

Comment:
Okay, I see. Could you please add a comment about it? I feel like it's not trivial.